### DISCUSSION TITLE
Notification for release from followed artists

### DISCUSSION THREAD

**Discussion by NikunjK8** [fast4x/RiMusic#1906] [General]
Is it possible to implement? SimpMusic added this feature, here's the commit:- https://github.com/maxrave-dev/SimpMusic/commit/8fb55120257d95f85ca9311cb21ed16f6a11b30d

**Comment by fast4x** [comment]
![Screenshot_2024-05-18-07-49-08-122_it.fast4x.rimusic.debug-edit.jpg](https://github.com/fast4x/RiMusic/assets/1484476/d54092c8-2c86-4621-a99e-427e82be6141)

![Screenshot_2024-05-18-07-48-41-876_it.fast4x.rimusic.debug-edit.jpg](https://github.com/fast4x/RiMusic/assets/1484476/7e21e0c4-a87d-45ef-b9ed-26d55dfc3dc2)

**Comment by NikunjK8** [comment]
Push notifications
For any release not just albums.

**Reply by fast4x** [reply]
mabe it require login ...

**Reply by NikunjK8** [reply]
I see. Check if youtube api rimusic is using support it otherwise drop the idea. 👍🏼
